common/app/views/fragments/javaScriptLaterSteps.scala.html

```diff
@@ -283,13 +283,13 @@
                 ab.run();
                 if(guardian.config.page.isFront) {
                     lazyLoadImages.init();
-                    images.upgradePicture();
+                    images.upgradePictures();
                     if(!document.addEventListener) { // IE8 and below
-                        window.onload = images.upgradePicture;
+                        window.onload = images.upgradePictures;
                     }
                 } else {
                     lazyLoadImages.setUpListeners();
-                    images.upgrade();
+                    images.upgradePictures();
```

Comment:
Given this happens on either branch, might as well just move it outside the if statement.